### DISCUSSION TITLE
Newsletter Interview

### DISCUSSION THREAD

**Discussion by ph4ni** [johnfactotum/foliate#1197] [General]
Hi John!, I'm Sai from [Console](https://console.substack.com/archive). We have featured Foliate previously in our newsletter, and we would love to have you for an interview. Let me know if you are interested in doing a text-based interview! My email is sai at codesee.io :)

**Comment by johnfactotum** [comment]
Ask away :)

**Comment by ph4ni** [comment]
Can you share your email? If you don't want to share it publicly, you can just send a hi to my email (mentioned in the above message)

**Reply by johnfactotum** [reply]
Very well. I've sent you an email.
